modules/mobile/spec/request/user_request_spec.rb

```diff
@@ -230,230 +231,6 @@
           )
         end
       end
```

Comment:
One of the major benefits of moving authorized services out of the serializer is that we're able to unit test, so we don't have to have these clunky tests anymore.